package.json

```diff
@@ -1,73 +1,28 @@
 {
     "private": true,
-    "workspaces": [
-        "src/*/assets"
-    ],
+    "workspaces": ["src/*/assets"],
     "scripts": {
         "build": "node bin/build_javascript.js && node bin/build_styles.js",
         "test": "bin/run-vitest-all.sh",
```

Comment:
Any impact on those two ? (build / test)

Reply:
Mmmh nope, it is not related to Biome and they still works